nautobot/dcim/choices.py

```diff
@@ -821,6 +821,17 @@ class InterfaceTypeChoices(ChoiceSet):
     TYPE_E1 = "e1"
     TYPE_T3 = "t3"
     TYPE_E3 = "e3"
+    TYPE_EIA232= "eia232"
```

Comment:
Would it be more self-consistent and make better sense for these to be the connector type rather than the protocol? E.g. `ConsolePortTypeChoices` has `TYPE_DE9 = "de-9"` and `TYPE_DB25 = "db-25"` rather than `eia232`.

Reply:
For serial port there is 4 elements:
* The physical connector, there is a lot.
* Is the connector a point to point, og bus.
* OSI model 1, how bits are send, EIA233, EIA485, BELL202, fiber,
* OSI model 2-6 is not used.
* OSI model 7, how the data is formatted, Profibus, MODbus, HART, and you can make your own.

So based on that i think a mix of OSI L1 and OSI L7, is what will  give us the best result, this is most inline with the other elements in the table.

And option is to have physical connector, as it own table.

Reply:
Tagging @lampwins to provide guidance here.

Reply:
Let's focus on the physical type and a custom field can be used for transport protocol where appropriate. In the future we can look at adding additional context to the data model here.